src/tools/tidy/src/style.rs

```diff
@@ -210,6 +210,7 @@ fn skip_markdown_path(path: &Path) -> bool {
         "src/doc/reference",
         "src/doc/rust-by-example",
         "src/doc/rustc-dev-guide",
+        "RELEASES.md",
```

Comment:
Why is RELEASES not ready for tidy?

Reply:
the file is too long and contains some `dbg!`s. There is no other way to disable it (e.g. comments or `tidy-ignore` in the filename).